homeassistant/components/watergate/strings.json

```
@@ -17,5 +17,54 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "water_meter_volume": {
+        "name": "Water Meter Volume"
+      },
+      "water_meter_duration": {
+        "name": "Water Meter Duration"
+      },
+      "ip": {
+        "name": "IP Address"
+      },
+      "gateway": {
+        "name": "Gateway Address"
+      },
+      "subnet": {
+        "name": "Subnet"
+      },
+      "ssid": {
+        "name": "SSID"
+      },
+      "rssi": {
+        "name": "RSSI"
+      },
+      "wifi_uptime": {
+        "name": "WIFI Uptime"
```

Wi-Fi is typically used

```suggestion
        "name": "Wi-Fi Uptime"
```